packages/storybook/src/nimble/text-field/text-field-matrix.stories.ts

```diff
@@ -161,7 +161,7 @@ const requiredVisibleStatesComponent = (
     </${textFieldTag}>
 `;
 
-export const lightThemeEditableEnabledWithoutButtons: StoryFn = createFixedThemeStory(
+export const lightTheme$Editable$Enabled$WithoutButtons: StoryFn = createFixedThemeStory(
```

Comment:
I feel like having `editable` and `enabled` in the names end up more confusing. If the state is unset maybe we should just not have that part in the name:

`lightTheme$Editable$Enabled$WithoutButtons` vs `lightTheme$WithoutButtons`

We kinda did that with appearanceReadOnly already. When unset it's just not in the name, we don't have a named value for it like AppearanceReadOnlyAbsent. So it would be doing that consistently and leaving unset parts out of the name

Reply:
Hmm looking again I guess `without buttons` would be removed from my example (we treat it binary and it's literally an absence of buttons).

Alternatively maybe one could argue we should have a name to represent `AppearanceReadOnlyAbsent` so that way adding a dimension is easier, count the number of `$` to know what part of the state you are editing consistently. 

I don't think I feel strongly either way, it's only coming up because the text field is extra complex / has so many states

Reply:
My leaning is to rename these stories in the text-field-matrix to include all parts of the state. At least in this case where we systematically go through all permutations of readonly on/off, disabled on/off, appearanceReadOnly on/off, and buttons on/off, keeping each name explicit about a state being on or off makes sense.